src/CLI/clitree/cli-xml/tam.xml

```diff
@@ -92,7 +92,7 @@ limitations under the License.
              help="Collector name" 
              ptype="STRING"> 
       </PARAM>
-      <PARAM name="ip-type" 
+      <PARAM name="type" 
```

Comment:
ip-addr also needs to changed to "ip" , in line 104, As the correct comamnd is "collector <collector-name> type <ipv4/ipv6> ip <ipaddress> port <collector-port>"

Reply:
Done